src/routes/api.ts

```diff
@@ -12,18 +19,64 @@ router.get('/', (req: Request, res: Response) => {
 });
 
 //Alive check
+//Status check
 router.get('/status', (req, res) => {
-  res.status(200).send("Alive");
+  res.status(GetServerStatus().statusCode).send(GetServerStatus().statusString);
 });
 
+
+router.get('/httpTest', (req,res)=>{
```

Comment:
Should be in it's own route since /api is a template / temporary.

Reply:
API should be removed, since it was originally only meant to be used for testing and simulating, in helping development of the "/status"-API.

I just forgot to remove it when I committed and pushed.